libraries/ms-drivers/test/test_drv120_relay.c

```diff
@@ -44,3 +64,16 @@ void test_drv120_relay(void) {
   TEST_ASSERT_OK(drv120_relay_get_is_closed(&closed));
   TEST_ASSERT_FALSE(closed);
 }
+
+void test_callback_registered(void) {
+  LOG_DEBUG("Initializing gpio interrupt\n");
+  Drv120RelaySettings settings = {
+    .pin = &s_test_drv120_pin,
+    .status = &s_test_drv120_status,
+    .handler = prv_callback,
+    .context = NULL,
```

Comment:
Might also want to assign an arbitrary non-null context and test that it was received so you know that contexts are passed correctly.